docroot/themes/custom/mass_theme/templates/content/node--topic-page.html.twig

```diff
@@ -103,18 +103,22 @@
   {# TOPIC PAGES show organizations and related topics at the bottom of page content as a bulleted list. #}
   {# Other content types show organizations as wrapped links at the top in the page header region. #}
   {# See: https://jira.mass.gov/browse/DP-13009 #}
-  <section class="ma__related-orgs-and-topics">
-    {% include "@atoms/04-headings/comp-heading.twig" with {
-      "compHeading": {
-        "title": "Related Information",
-        "level": level
-      },
-    } %}
-    <div class="ma__related-orgs-and-topics__columns">
-      {{ content.field_organizations }}
-      {{ content.field_topic_ref_related_topics }}
-    </div>
-  </section>
+  {% set related_tipics_are_visible = node.field_organizations.isEmpty != true or node.field_topic_ref_related_topics.isEmpty != true %}
```

Comment:
@dstorozhuk Sorry, missed this until after merge, but we can do a quick follow up down the road. Just a misspelling of "topics," but looks like it was consistently referenced elsewhere. So, low priority.